.gitignore

```diff
@@ -0,0 +1,33 @@
+# Logs
```

Comment:
OK，get it. However, the implement of node-sphericalmercator still have some problem. Please check out here: https://github.com/mapbox/node-sphericalmercator/issues/22